### PR TITLE
Enable node e2e accounting on systemd

### PR DESCRIPTION
**Pull request by derekwaynecarr**
Updated the e2e setup.sh script to enable cpu and memory accounting.

Related to https://github.com/kubernetes/kubernetes/issues/26198

/cc @pwittrock

### CODE REVIEW

**Comment by derekwaynecarr** [comment]
@pwittrock -  i don't quite know when in the flow or if this script is invoked or not to know how to respond to the pr checks... please assist.

File: test/e2e_node/environment/setup_host.sh [hunk 1]
```diff
@@ -25,6 +25,18 @@
 # - centos 7
 # - debian jessie
 
+# Enable CPU and memory accounting
+# tested rhel 7 (3.10.0-229.7.2.el7.x86_64) using systemd 219
+if [ -d /etc/systemd ]; then
+  sudo mkdir -p /etc/systemd/system.conf.d/
+  sudo cat <<EOF >/etc/systemd/system.conf.d/kubernetes-accounting.conf
```
Comment (derekwaynecarr) [inline test/e2e_node/environment/setup_host.sh]: @ingvagabund on rhel 7 with systemd 219, i had to use the above path.

**Comment by pwittrock** [comment]
This script is only invoked manually as part of the manual process of creating an image.  Your changes should not have caused any failures.  These are most likely flakes.

**Comment by pwittrock** [comment]
@k8s-bot e2e test this issue: #26270

**Comment by pwittrock** [comment]
@k8s-bot node e2e test this issue: #25993

**Comment by pwittrock** [comment]
@derekwaynecarr If you have not already, the best way to test that this script does what you want is to create a new rhel host, run the script on the pristine host and then run the node e2e test suite against that host using the comment in [e2e-node-jenkins.sh](https://github.com/kubernetes/kubernetes/blob/master/test/e2e_node/jenkins/e2e-node-jenkins.sh#L18) with your *.properties file (copied from the template.properties)

**Comment by pwittrock** [comment]
@derekwaynecarr Let me know if you are satisfied with any testing you have done using this script to setup the rhel host.

**Comment by derekwaynecarr** [comment]
@pwittrock - I was out of office most of yesterday, testing this now on GCE vanilla rhel 7 image.

**Comment by pwittrock** [comment]
@derekwaynecarr SGTM.  Wanted to make sure I wasn't blocking you.

**Comment by derekwaynecarr** [comment]
* Tested script on rhel, centos, debian jessie
* Updated how docker is installed on RHEL to use RHEL packages

/cc @pwittrock @vishh

**Comment by vishh** [comment]
@pwittrock @derekwaynecarr Are we adding RHEL and CentOS nodes to our node test pool?

**Comment by pwittrock** [comment]
We should if this makes it so they pass consistently.

**Comment by derekwaynecarr** [comment]
@pwittrock @vishh - I don't know what test pools you run internally, but the goal here was to just get the script updated to enable accounting globally however you guys setup the base images.  If you run this script on any systemd system, it should do the right thing for you and let us unblock other PRs.

CI stack for Red Hat stuff can be invoked on a per PR basis by just saying:

@redhat-k8s-bot test this

**Comment by pwittrock** [comment]
I am going to setup an instance with this script, and then lgtm it

**Comment by pwittrock** [comment]
@derekwaynecarr I tried to verify this today but ran into some trouble.  Will pick it up again tomorrow.

**Comment by pwittrock** [comment]
@k8s-bot e2e test this issue: #26210

[artifacts](https://console.cloud.google.com/storage/browser/kubernetes-jenkins/pr-logs/pull/26289/kubernetes-pull-build-test-e2e-gce/42992/)

**Pull request merged**